project.godot

```diff
@@ -17,6 +17,27 @@ run/main_scene="res://scenes/StartMenu.tscn"
 
 Game="*res://scripts/Game.gd"
 
+[importer_defaults]
+
+texture={
+"compress/hdr_mode": 0,
+"compress/lossy_quality": 0.7,
+"compress/mode": 0,
+"compress/normal_map": 0,
+"detect_3d": false,
+"flags/anisotropic": false,
+"flags/filter": true,
```

Comment:
I think you don't want this on in pixel art games. Afaik it maps to linear texture filtering, meaning the "pixel" boundaries will be blurry.